spec/features/forms/form/form_status_and_version_spec.rb

```diff
@@ -113,4 +113,41 @@
     click_link("Yourform")
     expect(page).to have_css("div#status", text: /Status\n Live/)
   end
+
+  scenario "changing version when live should enqueue the render form xml job for odk" do
+    visit(forms_path(mode: "m", mission_name: get_mission.compact_name, locale: "en"))
```

Comment:
weirdly this spec is failing with "mission not found", i'll look into it before merge